spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -92,6 +92,9 @@ trait CommonJobExecution extends ProjectMetadata {
         throw new IllegalStateException("Dataset validation failed, errors found in fields:\n" +
           validation.errors.map { case (field, errMsg) => s" - '$field': $errMsg" }.mkString("\n")
         )
+      case Some(validation) if validation.nonEmpty =>
+        log.warn("Dataset validation had some warnings:\n" +
+          validation.warnings.map { case (field, warnMsg) => s" - '$field': $warnMsg" }.mkString("\n"))
```

Comment:
After fixing, this turns out not very nice:
```
21/07/11 18:57:41 WARN StandardizationAndConformanceJob$: Dataset validation had some warnings:
 - 'Color': List(Property 'Color' is recommended to be present, but was not found!)
 - 'SNOW_CORRELATION_ID': List(Property 'SNOW_CORRELATION_ID' is recommended to be present, but was not found!)
```
```suggestion
          validation.warnings.map { case (field, warnMsg) => {
            val header = s" - '$field': "
            s"$header${warnMsg.mkString(s"\n$header")}" }.mkString("\n")}
        )
```